src/api/index.ts

```diff
@@ -0,0 +1,67 @@
+import express, { Router } from 'express';
+
+import { Routes } from '../constant';
+
+import * as cManifest from './manifest';
+import * as cBindings from './bindings';
+import * as cInstall from './install';
+import * as cConfigure from './configure';
+import * as cHelp from './help';
+import * as cAlert from './alert';
+import * as cTeam from './team';
+import * as cSubscription from './subscription';
+import * as cWebhook from './webhook';
+
+const router: Router = express.Router();
+
+router.get(Routes.App.ManifestPath, cManifest.getManifest);
+router.post(Routes.App.BindingsPath, cBindings.getBindings);
+router.post(Routes.App.InstallPath, cInstall.getInstall);
+
+router.post(`${Routes.App.CallPathHelp}`, cHelp.getHelp);
```

Comment:
Can we format each of the routes in this file like this instead?

```suggestion
router.post(Routes.App.CallPathHelp, cHelp.getHelp);
```